### ISSUE TITLE
Would it be possible to install APT package manager on ZimaOS?

### ISSUE THREAD

**Issue by serycoola** [IceWhaleTech/ZimaOS#117]
Even if as an experimental feature, on the users own risk? Some people require certain drivers that didn't come with the built in kernel...

**Comment by ProtocolNebula** [comment]
I cannot even install neither tmux without using docker. I don't understand why there is no package manager, or at least, there is no information about which one is...

**Comment by cosminsstoica** [comment]
strange, casaos has apt. zimaos was build on casaos... so

**Comment by cosminsstoica** [comment]
Well, I find the answer...



![image](https://github.com/user-attachments/assets/a33002de-12d9-443f-bd5c-f68d1a3287a6)


![image](https://github.com/user-attachments/assets/da86e50e-a52f-4429-9658-86366b08d75e)

**Comment by ProtocolNebula** [comment]
Thanks @cosminsstoica , this is like a hit in the bottom part...

But at least, issue solved, so is mandatory to have a docker running to run the desired commands... this does some tasks harder, but of course, secured...

**Comment by mistersupri** [comment]
yes, this kinda sucks... so many things that I need to do with package manager. even if there is is no package manager, this os should have brought basic package like openssl, wakeonlan, etc. I can't even make a cron job here without using service. I'm also having a hard time to install radarr on external storage (it said read only system) and coolify which I need to add authorized keys in root folder.
